app/views/accounts/_pii.html.erb

```diff
@@ -0,0 +1,99 @@
+<% if locked_for_session %>
+  <div class='mb2'>
+    <div class='usa-alert usa-alert--warning'>
+      <div class='usa-alert__body'>
+        <div class='usa-alert__text'>
+          <%= t('account.re_verify.banner') %>
+
+          <%= link_to(t('account.re_verify.footer'), user_password_confirm_path) %>
+        </div>
+      </div>
+    </div>
+  </div>
+<% end %>
+<div class='mb3 profile-info-box'>
+  <div class='bg-lightest-blue py1 px2 h6 caps clearfix'>
+    <div class='col col-11'>
+      <%= t('headings.account.profile_info') %>
+      <%= image_tag asset_url('user.svg'), width: 8, class: 'ml1' %>
+    </div>
+    <div class='col col-1 right-align'>
+      <%= image_tag asset_url('lock.svg'), width: 8, class: 'mr1' %>
+    </div>
+  </div>
+  <div class='p2'>
+    <div class='clearfix mxn1'>
+      <div class='sm-col sm-col-4 px1 bold'>
+        <%= t('account.index.full_name') %>
+      </div>
+      <div class='sm-col sm-col-8 px1 truncate'>
+        <%= pii.full_name %>
+      </div>
+    </div>
+  </div>
+  <div class='p2 border-top'>
+    <div class='clearfix mxn1'>
+      <div class='sm-col sm-col-4 px1 bold'>
+        <%= t('account.index.address') %>
+      </div>
+      <div class='sm-col sm-col-8 px1'>
+        <%= pii.address1 %>
+        <br>
+        <% if pii.address2.present? %>
+          <%= pii.address2 %>
+          <br>
+        <% end %>
+        <% if pii.city.present? && pii.state.present? %>
+          <%= "#{pii.city}, #{pii.state} #{pii.zipcode}" %>
+        <% end %>
```

Comment:
I'm just noticing it because it's in the same diff, so we don't need to change this now if you don't want to, but I bet that we could use the `shared/_address` partial here if we wanted to, since the PII object is a struct which allows hash attribute access? we could just pass in the PII as if it were an address

Reply:
This template has the `present?` check, but `shared/_address` doesn't. It seems like it should be there either way so I added the check to the shared template and re-used it in this `_pii` one.

Reply:
Ah good catch!